apps/src/componentLibrary/checkboxDropdown/CheckboxDropdown.story.tsx

```diff
@@ -0,0 +1,280 @@
+import React, {useState, useCallback} from 'react';
+import CheckboxDropdown, {CheckboxDropdownProps} from './index';
+import {Meta, Story} from '@storybook/react';
+
+export default {
+  title: 'DesignSystem/Checkbox Dropdown Component',
+  component: CheckboxDropdown,
+} as Meta;
+
+//
+// TEMPLATE
+//
+// This is needed to fix children type error (passing string instead of React.ReactNode type)
```

Comment:
Can you elaborate? What is the issue here?

Reply:
Oh, thanks for the catch!
That's some template code following me from some old components. Don't remember the initial issue, but deleted this line now and looks like everything is working ok. Deleted.